src/print.rs

```diff
@@ -187,6 +190,7 @@ impl BacktraceFrameFmt<'_, '_, '_> {
         symbol_name: Option<SymbolName<'_>>,
         filename: Option<BytesOrWideString<'_>>,
         lineno: Option<u32>,
+        colno: Option<u32>,
```

Comment:
This is unfortunately breaking a public-facing API, so could this add a new function instead (like `print_raw_with_column`) and have this function delegate to that one?